nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/layers/convolution/Conv1D.java

```diff
@@ -65,6 +65,14 @@ public Conv1D(@NonNull INDArray input, @NonNull INDArray weights, INDArray bias,
         this(wrapFilterNull(input, weights, bias), wrapOrNull(output), config);
     }
 
+    public Conv1D(INDArray input, INDArray weights, INDArray bias, Conv1DConfig conv1DConfig) {
+        this(wrapFilterNull(input, weights, bias), null, conv1DConfig);
+    }
+
+    public Conv1D(INDArray input, INDArray weights, Conv1DConfig conv1DConfig) {
+        this(wrapFilterNull(input, weights), null, conv1DConfig);
+    }
```

Comment:
With the dl4j-dev-tools PR, we should only need the first constructor.
We should put Lombok `@NonNull` annotation on non-optional args (input, weights) too for new constructors.
(I know we're missing the `@NonNull` on other constructors, we'll fix that eventually...)

Same thing for Conv2D, Conv3D, etc below.